cross-dev/Cargo.lock

```diff
@@ -0,0 +1,621 @@
+# This file is automatically @generated by Cargo.
```

Comment:
we should not have multiple lock files imo, instead `cross-dev` should be included in the root workspace, preferably also then without being in the default-members

Reply:
no, sadly you can't. :/

you can't `use bin;` if `bin` is a binary target, you need to have a `lib.rs` to use it in other crates, even if those crates are in the same workspace